using_images/s2i_images/php.adoc

```diff
@@ -124,6 +124,13 @@ The following environment variable sets its equivalent property value in the opc
 |16M
 |===
 
+ifdef::openshift-origin[]
+|`*OPCACHE_REVALIDATE_FREQ*`
+|How often to check script timestamps for updates, in seconds. 0 will result in OPcache checking for updates on every request.
```

Comment:
Seems fine to link to the PHP docs for more. You can also remove the ifdef here if this is just going to get picked up in OSE normally in the next release; docs team will make sure it gets in at the right time.